.gitignore

```diff
@@ -2,6 +2,7 @@
 .DS_Store
 /local
 /.vscode
+/docs_build
```

Comment:
Careful that what you push is actually part of _this_ PR ;).

Reply:
Though feel free to ignore this as we both know its coming in soon anyway

Reply:
Including now, but yes, it should be part of the `update_package_metadata` PR